tests/cli-tests/cltools/zstdgrep.sh.exit

```diff
@@ -0,0 +1 @@
+1
```

Comment:
Is this file intentional ?

Reply:
Yup, it is saying that the exit code should be 1.